lib/matplotlib/cbook/__init__.py

```diff
@@ -13,6 +13,12 @@
 from six.moves import xrange, zip
 from itertools import repeat
 import collections
+
+try:
+    from collections.abc import MappingView
+except ImportError:
+    MappingView = None
```

Comment:
I vote for using collections.MappingView directly then as I think the if MappingView = None is clunky